src/redux/DiaryCardSlice/DiaryCardSlice.ts

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from "@reduxjs/toolkit";
+import { DiaryData } from "../../types/DiaryCardData";
+
+interface DiaryCardState {
+  diaryCardEntries: DiaryData[];
+}
+
+const initialState = {
+  diaryCardEntries: [],
+} as DiaryCardState;
```

Comment:
this is not the correct way to add types use initialState: DiaryCardState like this